src/app/components/BibPage/BibDetails.jsx

```diff
@@ -72,6 +74,32 @@ const getDefinitionObject = (bibValues, fieldValue, fieldLinkable) => {
   );
 };
 
+const getOwner = (bib) => {
+  const items = bib.items;
+  const ownerArr = [];
+  let owner;
+
+  if (!items || !items.length) {
+    return null;
+  }
+
+  items.forEach(item => {
+    const ownerObj = item.owner && item.owner.length ?
+      item.owner[0].prefLabel : undefined;
+
+    ownerArr.push(ownerObj);
+  });
+
+  if (_every(ownerArr, (o) => (o === ownerArr[0]))) {
```

Comment:
This is checking to see if the first item in the array matches every other item in the array. If it does, then the next step is to see if it's a non-NYPL bib and then we go from there.

But yes, if a bib and its items belong to NYPL, we don't show that. But if there are some items that belong to NYPL and some items that belong to someone else, we show nothing. If ALL the names the same (and non-nypl) we display it.